.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # Global rule:
-*                         @expressvpn-pete-m @expressvpn-tom-l @expressvpn-raihaan-m @expressvpn-andre-l
+*                         @expressvpn-pete-m @expressvpn-tom-l @expressvpn-raihaan-m @expressvpn-andre-l @expressvpn-mariappan-r @xv-ian-c
```

Comment:
I believe Maari would like to use @kp-mariappan-ramasamy now, see https://github.com/expressvpn/wolfssl-rs/blob/main/.github/CODEOWNERS

Reply:
Done.

Reply:
Thanks @expressvpn-andre-l @expressvpn-tom-l